src/client/routes/index.tsx

```diff
@@ -49,16 +50,22 @@ const routesMap = [
 export const Routes = () => {
   return (
     <Router basename="/#">
-      <Layout>
-        <Switch>
-          {routesMap.map((route, index) => (
-            <Route key={index} exact path={route.path} component={route.component} />
-          ))}
-          <Route path="*">
-            <Redirect to="/home" />
-          </Route>
-        </Switch>
-      </Layout>
+      <Switch>
+        <Route exact path="/health" component={Health} />
```

Comment:
is there a reason why health cannot be on the routesMap?

Reply:
A health route should contain minimal content as possible so it doesnt take much server resources since we might be consuming this route frequently in our monitor system. Did not include it in the routesMap since it renders element inside the Layout component.